lms/envs/common.py

```diff
@@ -2323,6 +2323,7 @@
 }
 
 # E-Commerce API Configuration
+ECOMMERCE_EXTERNAL_URL_ROOT = None
```

Comment:
What is the difference between this one and the ECommerce API URL?  Is this meant to be like the lms or cms BASE_URL's?